tests/ui/traits/const-traits/cross-crate.stocknc.stderr

```diff
@@ -1,3 +1,9 @@
+error[E0277]: the trait bound `cross_crate::NonConst: ~const cross_crate::MyTrait` is not satisfied
+  --> $DIR/cross-crate.rs:19:5
+   |
+LL |     NonConst.func();
+   |     ^^^^^^^^^^^^^^^
+
```

Comment:
note that when we constify libcore in the future, this change may be leaking out unstable details (the `~const` syntax) on stable code